File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.java
class Solution {
    public String[] solution(String[] quiz) {
        String[] answer = new String[quiz.length];
        
        for(int i = 0; i < quiz.length; i++){
            String[] str = quiz[i].split(" ");
            
            int X = Integer.parseInt(str[0]);
            String oper = str[1];
            int Y = Integer.parseInt(str[2]);
            int Z = Integer.parseInt(str[4]);
            
            if(oper.equals("+")){
                answer[i] = X + Y == Z ? "O" : "X";
            }else {
                answer[i] = X - Y == Z ? "O" : "X";
            }
        }
        
        return answer;
    }
}

File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.py
def solution(quiz):
    answer = []
    
    for q in quiz:
        element = q.split(' ');
        op = element[1]
        X = int(element[0])
        Y = int(element[2])
        Z = int(element[4])

        if op == '-':
            answer.append("O") if X - Y == Z else answer.append("X")
        elif op == '+':
            answer.append("O") if X + Y == Z else answer.append("X")

    return answer